app/notifications/process_notifications.py

```diff
@@ -96,7 +96,6 @@ def persist_notification(
         template_version=template_version,
         to=recipient,
         service_id=service.id,
-        service=service,
```

Comment:
I suspect this is going to break things, for instance:
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/16893311/acb073a2-8b4c-407d-b6b4-6fcbf2594f26)

We need to ensure this does not create additional breaking changes with these calls:
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/16893311/a99d18e2-8c79-4226-b374-74894cdf446f)

Reply:
1. That service is created using writer db and full service model, so it should be fine until we start changing that functionality.
2. The information that was written as part of persist_notification was just fetched with a service and is unchanged in the process. 
3.  Service_id is the required field because it's a foreign key, but not service. We can think of service_id and service as unrelated in the sense that they serve different purposes, but they are typically used together. Service_id is a column that physically stores the relationship between records and is used at the database level to enforce referential integrity. Service is an SQLAlchemy abstraction that provides convenient access to the related service object. It doesn't affect the database schema directly but tells the ORM how to load related objects. We don't have to have the service object as part of the notification object when we want to persist data.
 
![Screen Shot 2023-08-15 at 5 20 22 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/2191516/b5d9f6fb-94c1-4524-a38f-8a8302f8fda5)

![Screen Shot 2023-08-15 at 5 22 16 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/2191516/f52cbeca-3ac9-43c7-832e-f2d9504d7675)

![Screen Shot 2023-08-15 at 6 06 06 AM](https://github.com/department-of-veterans-affairs/notification-api/assets/2191516/35824d35-0b3d-49cb-8d0c-a7eac5e82694)

Reply:
Since `service_id` is a FK on the services table, the relationship will use that service id to lazy query for service when someone calls notification.service? The flask-sqlalchemy docs are [pretty light](https://flask-sqlalchemy.palletsprojects.com/en/3.0.x/models/) on the topic.